src/main.rs

```diff
@@ -157,3 +123,24 @@ fn ban<'m>(args: Args<'m>) -> Result {
     }
     Ok(())
 }
+
+/// Write the welcome message to the welcome channel.  
+fn welcome_message(args: Args) -> Result {
+    const WELCOME_BILLBOARD: &'static str = "By participating in this community, you agree to follow the Rust Code of Conduct, as linked below. Please click the :white_check_mark: below to acknowledge, or otherwise you may leave this Discord.
+
+  https://www.rust-lang.org/policies/code-of-conduct  ";
```

Comment:
Do we want this to embed? If not, wrap it in angular brackets. `<link>` will prevent `link` from creating an embed. Idk what the embed looks like, so your call.